charts/opentelemetry-collector/README.md

```diff
@@ -113,7 +136,7 @@ it is easy to accidentally feed the exported logs back into the receiver.
 Also note that using the `--verbosity=detailed` option for the `debug` exporter causes it to output
 multiple lines per single received log, which when looped, would amplify the logs exponentially.
 
-To prevent the looping, the default configuration of the receiver excludes logs from the collector's containers.
+To prevent the looping, the default configuration of the receiver excludes logs from the collector's containers by setting `presets.logsCollection.includeCollectorLogs` to `false`.
```

Comment:
```suggestion
To prevent the looping, the default configuration of the receiver excludes logs from the collector's containers.
```

I don't think it is necessary to callout how the preset is excluding the collector's logs.